modules/api/php/endpoints/candidate/visit/visit.class.inc

```diff
@@ -215,11 +215,19 @@ class Visit extends Endpoint implements \LORIS\Middleware\ETagCalculator
             $this->_candidate->getListOfVisitLabels()
         );
 
+        if (!in_array($this->_candidate->getCenterID(), $user->getCenterIDs())) {
```

Comment:
The `Candidate` class implements the `AccessibleResource` interface. I think this should just be using`$this->_candidate->isAccessibeBy($user)` instead of duplicating the logic in slightly different ways in the API. (That would also handle Project permissions, which this is currently not doing.)

Reply:
It is now using $this->_candidate->isAccessibleBy($user)